common/config/server-config.json

```diff
@@ -18,5 +18,5 @@
 		"maxDuplicates": 3,
 		"maxDeckCost": 42
 	},
-	"logoSubText": "Tournament Time!"
+	"logoSubText": "中國加油!"
```

Comment:
Is this an intentional change for this PR?

Reply:
Nope, was just experimenting with how non latin characters are displayed